geo/benches/triangulate.rs

```diff
@@ -0,0 +1,41 @@
+use criterion::{criterion_group, criterion_main};
+use geo::algorithm::TriangulateEarcut;
+use geo::algorithm::TriangulateSpade;
+
+fn criterion_benchmark(c: &mut criterion::Criterion) {
+    c.bench_function("TriangulateSpade (unconstrained)", |bencher| {
+        let multi_poly = geo_test_fixtures::nl_zones::<f64>();
+        bencher.iter(|| {
+            for poly in &multi_poly {
+                let triangulation = TriangulateSpade::unconstrained_triangulation(poly).unwrap();
+                assert!(triangulation.len() > 1);
+                criterion::black_box(triangulation);
+            }
+        });
+    });
+
+    c.bench_function("TriangulateSpade (constrained)", |bencher| {
+        let multi_poly = geo_test_fixtures::nl_zones::<f64>();
+        bencher.iter(|| {
+            for poly in &multi_poly {
+                let triangulation = TriangulateSpade::unconstrained_triangulation(poly).unwrap();
```

Comment:
A friend of mine just pointed out that both benchmarks do the same thing since they both use the `unconstrained_triangulation` method